server/adaptors/integrations/__data__/repository/nginx/schemas/logs-1.0.0.mapping.json

```diff
@@ -219,8 +219,13 @@
       }
     },
     "composed_of": [
+<<<<<<< Updated upstream
       "http",
       "communication"
+=======
+      "http_template",
+      "communication_template"
```

Comment:
Merge markers -- Should just revert all the changes for `nginx`. `git checkout swiddis/osints/dev -- server/adaptors/integrations/__data__/repository/nginx`